DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: bigrquery
 Title: An Interface to Google's 'BigQuery' 'API'
-Version: 1.2.0
+Version: 1.2.0-1
```

Comment:
Would it be worth making it more obvious to our users that this is an AOU-specific version? Like `1.2.0-aou-1`?  I guess if the differences are this small it's not likely to be an issue.

Reply:
Valid characters are numbers, -, and .  http://r-pkgs.had.co.nz/description.html#version

Reply:
So we can use Morse code.